src/components/existing-request/existing-request-display.vue

```diff
@@ -228,6 +260,14 @@ export default class ExistingRequestDisplay extends Mixins(
   /** The pending payment, if any. See mounted(). */
   private pendingPayment = null
 
+  /** Params used to format refund message (tooltip and modal). */
+  private refundParams = {
```

Comment:
This object could probably benefit from an interface.

Reply:
👍